admin-dev/themes/new-theme/js/pages/product/combination/bulk.ts

```diff
@@ -37,4 +37,35 @@ $(() => {
   ]);
   new ImageSelector();
   new QuantityModeSwitcher();
+
+  const priceExcludedTaxId = 'bulk_combination_price_price_tax_excluded';
+  const priceIncludedTaxId = 'bulk_combination_price_price_tax_included';
+  const vatRate = 'bulk_combination_price';
+  const priceDiv: HTMLDivElement = document.getElementById(vatRate) as HTMLDivElement;
+  const priceExcludedTaxInput: HTMLInputElement = document.getElementById(priceExcludedTaxId) as HTMLInputElement;
+  const priceIncludedTaxInput: HTMLInputElement = document.getElementById(priceIncludedTaxId) as HTMLInputElement;
+  const rate: number = 1 + parseFloat((priceDiv.dataset.rate as string));
+
+  priceExcludedTaxInput.addEventListener('keyup', () => {
+    let value;
+
+    if (priceExcludedTaxInput.value === '') {
+      value = 0;
+    } else {
+      value = parseFloat(priceExcludedTaxInput.value);
+    }
+    priceIncludedTaxInput.value = (value * rate).toString();
```

Comment:
I wonder if we could use [this](https://github.com/PrestaShop/PrestaShop/blob/develop/admin-dev/themes/new-theme/js/pages/order/view/order-prices.ts) here, instead of making the calculation here

Reply:
Not sure about using this as we don't have precision needed and we can avoid a dependency to OrderPrice here.